components/Wizard/ImageStep.js

```diff
@@ -308,6 +311,41 @@ class ImageStep extends React.PureComponent {
       </FormGroup>
     );
 
+    const vmwareProviderCheckbox = (
+      <FormGroup
+        label={<FormattedMessage defaultMessage="Upload image" />}
+        labelIcon={
+          <Popover
+            id="popover-help"
+            bodyContent={
+              <FormattedMessage
+                defaultMessage="
+                      Image Builder can can upload images you create to VMWare vSphere.
+                      When the image build is complete and the upload action is successful,
+                      the image file is available in the Cluster on vSphere instance that you specified.
```

Comment:
Minor nitpick. Could you say "in the Cluster on `the` vSphere instance that you specified"

Reply:
Also could you make the statement "VMWare vSphere" a link to the VMWare vSphere.

Reply:
I was thinking about that link but AFAIK vSphere doesn't have a public instance similar to AWS or Azure, it's a product you privately so I didn't see much point in it, but I can link it to https://www.vmware.com/products/vsphere.html if you think it makes sense.

Reply:
Mmm. Okay. I wasn't aware of that. I think you're right that there isn't much of a point to link to the product page. Thanks for the clarification.